config/config.go

```diff
@@ -128,6 +128,9 @@ func (cfg *AnalyzerConfig) Validate() error {
 
 // ServerConfig contains the API server configuration.
 type ServerConfig struct {
+	// ChainID is the chain ID of the chain this analyzer will process.
```

Comment:
```suggestion
	// ChainID is the chain ID of the default (typically latest) chain ID for the server API.
```
something like this is more accurate -- this is now in the API layer not analysis

Reply:
Oops 😄 

Hmm, I think we don't set a default? I'll say that it is usually the latest.